docs/sources/configuration/examples/13-COS-Trusted-Profile-Example.yaml

```diff
@@ -0,0 +1,25 @@
+# This partial configuration uses IBM Cloud Object Storage (COS) for chunk storage. 
+# Trusted profile will be used for authenticating with COS. We can either pass
+# trusted profile name or trusted profile ID along with the compute resource token file.
+# If we pass both trusted profile name and trusted profile ID it should be of 
+# same trusted profile.
+
+schema_config:
+  configs:
+    - from: 2020-10-01
+      index:
+        period: 24h
+        prefix: loki_index_
+      object_store: "cos"
+      schema: v11
+      store: "boltdb-shipper"
+
+storage_config:
+  cos:
+    bucketnames: <bucket1, bucket2>
+    endpoint: <endpoint>
+    region: <region>
+    auth_endpoint: <iam_endpoint_for_authentication>
+    cr_token_file_path: <path_to_compute_resource_token>
+    trusted_profile_name: <name_of_the_trusted_profile>
```

Comment:
we need either the name or id, having both the name and id is confusing. Let's either use one of them, add a comment to explicitly mention that, or add another example for using the id instead of name.